tests/explorer.api.tests/TestWebAppFactory.cs

```diff
@@ -19,6 +19,7 @@ public class TestWebAppFactory : WebApplicationFactory<Startup>
     {
         public static readonly ExplorerConfig Config = new ConfigurationBuilder()
             .AddJsonFile("appsettings.Development.json")
```

Comment:
Should we use the `"appsettings.Test.json"`  here?
```suggestion
            .AddJsonFile("appsettings.Test.json")
```

Reply:
Yes, I missed that one!